server/resource_server/repository/repository_test.go

```diff
@@ -160,7 +159,17 @@ func TestLoginPreCheckUser(t *testing.T) {
 	}
 
 	// Expect a query to be executed and return a row
-	mock.ExpectQuery(regexp.QuoteMeta(`SELECT * FROM "users" WHERE username = $1 ORDER BY "users"."id" LIMIT 1`)).WithArgs(testLoginPrecheck.Username).WillReturnRows(sqlmock.NewRows([]string{"id", "email", "username", "first_name", "last_name", "password", "salt"}).AddRow(1, "user@test.com", "test_user", "Test", "User", "testpass", "testsalt"))
+	mock.ExpectQuery(
```

Comment:
Eventually you will be the owner of this function. May as well do these edits now 👍